src/components/registration-form.jsx

```diff
@@ -0,0 +1,122 @@
+        });
+    };
+
+    return (
+        <Row>
+            <Col>
+                <Formik
+                    validationSchema={schema}
+                    onSubmit={(values) => handleRegistration(values)}
+                    initialValues={{
+                        name: DEFAULT_REGISTRATION_NAME,
+                        email: DEFAULT_REGISTRATION_EMAIL,
+                        password: DEFAULT_REGISTRATION_PASSWORD,
+                    }}
+                >
+                    {({
+                        handleSubmit,
+                        handleChange,
+                        values,
+                        errors,
+                        isValid,
+                        isSubmitting,
+                    }) => (
+                        <Form noValidate onSubmit={handleSubmit}>
+                            <Container style={{ width: "50%" }}>
+                                <Row style={{ marginTop: "2rem" }}>
+                                    <h2>Register</h2>
+                                </Row>
+
+                                <Row style={{ marginTop: "2rem" }}>
+                                    <CustomFormTextField
+                                        label="Name"
+                                        name="name"
+                                    />
+                                </Row>
+
+                                <Row style={{ marginTop: "1rem" }}>
+                                    <CustomFormTextField
+                                        label="Email"
+                                        name="email"
+                                    />
+                                </Row>
+
+                                <Row style={{ marginTop: "1rem" }}>
+                                    <CustomFormTextField
+                                        label="Password"
+                                        name="password"
+                                    />
+                                </Row>
+
+                                <Row>
+                                    <Col>
+                                        <Button
+                                            disabled={!isValid || isSubmitting}
+                                            variant="success"
+                                            as="input"
+                                            size="lg"
+                                            type="submit"
+                                            value="Submit"
+                                            style={{
+                                                width: "10rem",
+                                                margin: "2rem",
+                                            }}
+                                        />
+                                    </Col>
+                                </Row>
+
+                                {/* ovde me interesuje kako mogu dinamicki da prosledim title i message npr. */}
```

Comment:
Ovo nisam siguran da razumem. Mozes umesto zakucanog stringa staviti samo neku promenljivu, konstantu kao sto to radis npr za `show` prop, ako si na to mislio 🤔